Tools/MiniBrowser/mac/AppDelegate.m

```diff
@@ -497,4 +497,25 @@ - (IBAction)clearDefaultStoreWebsiteData:(id)sender
     }];
 }
 
+- (void)websiteDataStore:(WKWebsiteDataStore *)dataStore domain:(NSString *)registrableDomain didOpenDomainViaWindowOpen:(NSString *)openedRegistrableDomain withProperty:(WKWindowProxyProperty)property directly:(BOOL)directly
+{
+    const char* propertyDescription;
+    switch (property) {
+    case WKWindowProxyPropertyInitialOpen:
+        propertyDescription = "initialOpen";
+        break;
+    case WKWindowProxyPropertyClosed:
+        propertyDescription = "closed";
+        break;
+    case WKWindowProxyPropertyPostMessage:
+        propertyDescription = "postMessage";
+        break;
+    default:
```

Comment:
I remember a policy to not have default cases. But maybe that's just in C++? I've been used to having an ASSERT_NOT_REACHED instead.